packages/lib/src/components/Card/Card.tsx

```diff
@@ -13,9 +13,12 @@ import { ClickToPayCheckoutPayload, IClickToPayService } from '../internal/Click
 import ClickToPayWrapper from './components/ClickToPayWrapper';
 import { UIElementStatus } from '../types';
 import SRPanelProvider from '../../core/Errors/SRPanelProvider';
+import { ThreeDS2Challenge, ThreeDS2DeviceFingerprint } from '../ThreeDS2';
 
 export class CardElement extends UIElement<CardElementProps> {
     public static type = 'scheme';
+    public static txVariants = ['amex', 'card', 'diners', 'discover', 'jcb', 'kcp', 'maestro', 'mc', 'scheme', 'storedCard', 'visa'];
```

Comment:
We need to do some research to find out if merchants ever actual go to do `checkout.create('mc')`, for example.
Because then we could thin down this list

Reply:
This list can, in fact, just be `['card']`. 
If merchants want to create a single branded card they can do so by just setting the `brands` config prop to contain a single txvariant